### PR TITLE
Final Competition Code

### PR DESCRIPTION
**Pull request by jack60612**
- [x] Test on Robot
- [ ] Hold Arm stable
- [ ] Hold Arm stable while moving

### CODE REVIEW

File: src/main/java/frc/robot/commands/ArmCommand.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
+
+  public void up() {
+    m_armSubsystem.armY(0.0, 0.0);
```
Comment (sviridovt) [inline src/main/java/frc/robot/commands/ArmCommand.java:42]: Lets not merge this until we can finalize these values

**Review by sviridovt — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -71,15 +106,60 @@ public RobotContainer() {
...
+            new PIDConstants(
+                DriveConstants.kPDriveVel, 0, 0), // A PID controller for wheel velocity control
+            m_driveSubsystem
+                ::tankDriveVolts, // A consumer that takes left and right wheel voltages and sets
+            // them to the drive subsystem's controllers
+            autonomousEventMap,
+            true, // change for either team
+            m_driveSubsystem //  Requirements of the commands (should be the drive subsystem)
+            );
   }
```
Comment (sviridovt) [inline src/main/java/frc/robot/RobotContainer.java:163]: Are we resetting the odometry before initializing? ref https://github.com/mjansen4857/pathplanner/wiki/PathPlannerLib:-Java-Usage#ppramsetecommand

File: src/main/java/frc/robot/DriveConstants.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  // Kinematic constants
+
+  // These characterization values MUST be determined either experimentally or theoretically
+  // for *your* robot's drive.
+  // The Robot Characterization Toolsuite provides a convenient tool for obtaining these
+  // values for your robot.
+  // Feed Forward Constants
+  public static final double Ks = 0.74056; // volts
+  public static final double Kv = 2.4474; // VoltSecondsPerMeter
+  public static final double Ka = 0.45481; // VoltSecondsSquaredPerMeter
```
[the remark this answers is not in the document]
Comment (jack60612) [reply src/main/java/frc/robot/DriveConstants.java:30]: i am going to re run the tests later today

**Review by sviridovt — APPROVED** [review]

**Comment by jack60612** [comment]
Yay well done @TimeToDie2 @sviridovt see you on Monday!

**Pull request merged**